### PR TITLE
exc_1 Done

### PR DESCRIPTION
**Pull request by hadeel-ghourani**

### CODE REVIEW

File: exc_1.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                               self.destination_location, 
+                               self.fuel, minimum_fuel_needed))
+        else:
+            print("The amount of fuel: ", self.fuel)
+            print("The amount of used fuel: ", distance * 4)
+            print("The coverd distance: ", distance * 4)
+            print("The remaining fuel: ", self.fuel - distance * 4)
+
+    def __str__(self):
+        print("Current Location: ", self.get_plam_location())
```
Comment (belal-b-ali) [inline exc_1.py]:
```suggestion
        print("Current Location: ", self.get_plane_location())
```

File: exc_1.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        print("Fuel: ", self.fuel())
```
Comment (belal-b-ali) [inline exc_1.py]: `fuel` is attribute it's not callable

File: exc_1.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        print(self.fly())
```
Comment (belal-b-ali) [inline exc_1.py]: why you are printing `fly` !?

File: exc_1.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
+class FuelShortage(Exception):
+    def __init__(self, current_location, destination_location, fuel, min_fuel):
+        self.current_location = current_location
+        self.destination_location = destination_location
+        self.current_fuel = fuel
+        self.min_fuel_needed = min_fuel
```
Comment (belal-b-ali) [inline exc_1.py]: using `self` is not needed here

**Review by belal-b-ali — CHANGES_REQUESTED** [review]

File: exc_1.py (hunk 1 above)
Comment (hadeel-ghourani) [reply exc_1.py]: Done..

**Review by belal-b-ali — APPROVED** [review]

**Pull request merged**